scripts.js

```diff
@@ -0,0 +1,88 @@
+function signup() {
+  var handle = document.getElementById("handle")
+  var password = document.getElementById("password")
+
+  var url = 'https://chitter-backend-api.herokuapp.com/users';
+  var data = {user: {handle: handle.value, password: password.value}};
+
+  fetch(url, {
+    method: 'POST',
+    body: JSON.stringify(data),
+    headers:{
+      'Content-Type': 'application/json'
+    }
+  })
+}
+
+function userLogIn() {
+  var handleSignin = document.getElementById("handleSignIn")
+  var passwordSignin = document.getElementById("passwordSignIn")
+
+  var urlSession = 'https://chitter-backend-api.herokuapp.com/sessions';
+  var dataSession = {session: {handle: handleSignin.value, password: passwordSignin.value}};
+
+  $.ajax({
+    url: urlSession,
+    type: 'post',
+    dataType: 'json',
+    contentType: 'application/json',
+    success: function (data) {
+        var logInDetails = data;
+        loadPeeps(logInDetails);
+    },
+    data: JSON.stringify(dataSession)
+  });
+};
+
+function newPeep(logInDetails) {
+  
+  var header = document.createElement("h3");
+  header.appendChild(document.createTextNode(logInDetails.user_id));
+  document.body.appendChild(header);
+  var x = document.createElement("textarea");
+  x.setAttribute("id", "peep")
+  document.body.appendChild(x);
+  var btn = document.createElement("BUTTON");
```

Comment:
I found this section with adding in all the html  bits a little bit hard to read, but maybe I just need to be a bit more familiar with pure JS.